data/json/furniture_and_terrain/furniture-seats.json

```
@@ -204,5 +204,218 @@
       "sound_fail": "whump.",
       "items": [ { "item": "2x4", "count": [ 1, 3 ] }, { "item": "nail", "charges": [ 2, 6 ] }, { "item": "splinter", "count": 1 } ]
     }
+  },
+  {
+    "type": "furniture",
+    "id": "f_floralprint_armchair",
+    "name": "floral print armchair",
+    "symbol": "H",
+    "description": "A more comfortable way of sitting down.  This reminds you of a nursing home.",
+    "looks_like": "f_armchair",
+    "color": "pink",
+    "move_cost_mod": 1,
+    "coverage": 45,
+    "comfort": 3,
+    "floor_bedding_warmth": 500,
+    "bonus_fire_warmth_feet": 1000,
+    "required_str": 7,
+    "deconstruct": {
+      "items": [ { "item": "2x4", "count": 10 }, { "item": "rag", "count": [ 30, 33 ] }, { "item": "nail", "charges": [ 6, 8 ] } ]
+    },
+    "max_volume": 4000,
+    "flags": [ "TRANSPARENT", "FLAMMABLE_ASH", "ORGANIC", "PLACE_ITEM", "MOUNTABLE", "CAN_SIT" ],
+    "bash": {
+      "str_min": 12,
+      "str_max": 40,
+      "sound": "smash!",
+      "sound_fail": "whump.",
+      "items": [
+        { "item": "2x4", "count": [ 1, 3 ] },
+        { "item": "nail", "charges": [ 2, 6 ] },
+        { "item": "splinter", "count": 1 },
+        { "item": "rag", "count": [ 20, 30 ] }
+      ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_beanbag_chair",
+    "name": "beanbag chair",
+    "symbol": "#",
+    "description": "A more comfortable way of sitting down.  It smells like junk food and incense.",
+    "color": "blue",
+    "move_cost_mod": 1,
+    "coverage": 35,
+    "comfort": 3,
+    "floor_bedding_warmth": 500,
+    "bonus_fire_warmth_feet": 1000,
+    "required_str": 7,
+    "deconstruct": { "items": [ { "item": "rag", "count": [ 25, 30 ] }, { "item": "cotton_ball", "charges": [ 2, 5 ] } ] },
+    "max_volume": 3000,
+    "flags": [ "TRANSPARENT", "FLAMMABLE_ASH", "ORGANIC", "PLACE_ITEM", "MOUNTABLE", "CAN_SIT", "SHORT" ],
+    "bash": {
+      "str_min": 8,
+      "str_max": 20,
+      "sound": "smash!",
+      "sound_fail": "whump.",
+      "items": [ { "item": "cotton_ball", "count": [ 2, 5 ] }, { "item": "rag", "count": [ 20, 30 ] } ]
```

```suggestion
      "items": [ { "item": "cotton_ball", "charges": [ 2, 5 ] }, { "item": "rag", "count": [ 20, 30 ] } ]
```

Assuming you meant it this way; as it is, it drops more cotton when smashed than when deconstructed.